completions/links

```diff
@@ -80,8 +80,10 @@ _comp_cmd_links()
     esac
 
     if [[ $cur == -* ]]; then
-        COMPREPLY=($(compgen -W '$(_parse_help "$1" |
-            command grep -vF -- "->")' -- "$cur"))
+        _comp_compgen -R help
+        ((${#COMPREPLY[@]})) &&
+            _comp_compgen -- -W '$(printf "%s\n" "${COMPREPLY[@]}" |
+                command grep -vF -- "->")'
```

Comment:
```suggestion
            _comp_compgen -- -W '"${COMPREPLY[@]}"' -X "->"
```

Reply:
Thank you for these suggestions! You are right, we can do them just by the builtin feature. I would like to apply these changes, but I'd like to confirm one thing here. The above `grep -vF -- "->"` is technically equivalent to `-X '*->*'`, but does the intent actually to remove the exact matching to `->`?

Reply:
Thank you. I have applied the changes.